jib-cli/src/integration-test/java/com/google/cloud/tools/jib/cli/JarCommandTest.java

```diff
@@ -331,4 +332,16 @@ private static String getContent(URL url) throws InterruptedException {
     }
     return null;
   }
+
+  private String getDockerHostForHttpRequest() {
```

Comment:
This method seems like a good candidate for a utility class. https://github.com/GoogleContainerTools/jib/tree/master/jib-plugins-common/src/test/java/com/google/cloud/tools/jib/api looks like a fitting home for this class.

Reply:
Thanks for the pointer @mpeddada1! For the utility class, I adapted and moved the maven plugin's [`HttpGetVerifier`](https://github.com/GoogleContainerTools/jib/blob/master/jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/HttpGetVerifier.java) helper to this location, since it contains the http request logic used by both plugins for integration testing. 

A quick question for jib-cli: I also made a copy of this test helper into https://github.com/GoogleContainerTools/jib/tree/master/jib-cli/src/test/java/com/google/cloud/tools/jib/api since I don't think the `jib-cli` tests should depend on `jib-plugins-common`, but do you have more context on the setup here and whether this refactoring makes sense?